src/robot_dart/control/pd_control.cpp

```diff
@@ -8,6 +8,8 @@ namespace robot_dart {
         PDControl::PDControl(const Eigen::VectorXd& ctrl, bool full_control) : RobotControl(ctrl, full_control) {}
         PDControl::PDControl(const Eigen::VectorXd& ctrl, const std::vector<std::string>& controllable_dofs) : RobotControl(ctrl, controllable_dofs) {}
 
+
+
```

Comment:
Why adding spaces here?

Reply:
Spaces are still here..

Reply:
These I am not sure I can find, can you point them out one more time ? Thank you

Reply:
@dinies do not forget this one ;)

Reply:
> These I am not sure I can find, can you point them out one more time ? Thank you

No worries. Leave them. I'll fix them with the rotation thing..